websockets/bridge.go

```diff
@@ -72,6 +24,11 @@ type ObservationPublisher interface {
 // and message routing, while delegating protocol-specific logic to the
 // provided message handlers.
 //
+// Architecture:
+// - Protocol-agnostic: Works with any protocol (Shannon, future protocols)
+// - Message handlers: Protocol-specific logic for signing/validation
+// - Notification channels: Gateway observability without tight coupling
+//
 // Full data flow: Client <---clientConn---> PATH bridge <---endpointConn---> Relay Miner bridge <------> Endpoint
 type bridge struct {
```

Comment:
Please consider revising the websockets package: it needs a clear entry point, to be utilized by the gateway package.